go/vt/vtcombo/tablet_map.go

```diff
@@ -742,8 +742,8 @@ func (itmc *internalTabletManagerClient) SlaveWasRestarted(ctx context.Context,
 	return fmt.Errorf("not implemented in vtcombo")
 }
 
-func (itmc *internalTabletManagerClient) StopReplicationAndGetStatus(ctx context.Context, tablet *topodatapb.Tablet) (*replicationdatapb.Status, error) {
-	return nil, fmt.Errorf("not implemented in vtcombo")
+func (itmc *internalTabletManagerClient) StopReplicationAndGetStatus(ctx context.Context, tablet *topodatapb.Tablet, stopReplicationOption string) (*replicationdatapb.Status, *replicationdatapb.StopReplicationStatus, error) {
```

Comment:
What does `StopReplicationOption` stand for? I only see references with empty strings; would it make sense to create a new `type` to clarify what the options are?

Reply:
I did actually create a new type on the server-side which I cast to. The issue I found was that at the gRPC barrier it has to be carried through as a normal string. That begs the question of where do we define this type? Anyone that needs to know this type must directly depend on where it's being called. On which side of the gRPC barrier should that type live, given that anyone who needs to know about it must depend upon it?

Reply:
I touched base with @enisoc offline and it looks like the established pattern to solve this issue in Vitess is to declare the enum in proto and depend on the generated code. So I'll update this now!

Reply:
Done